package.json

```diff
@@ -31,6 +32,7 @@
     "jest-junit": "^12.2.0",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
+    "react-recaptcha": "^2.3.10",
```

Comment:
[`react-recaptcha`](https://www.npmjs.com/package/react-recaptcha/v/2.3.10) doesn't seem to be actively maintained (last publish was 3 years ago), but if we don't run into any limitations we should be good.

Reply:
Hmm, well if we do,  react-google-recaptcha's last update was a year ago and that would be a pretty simple fix. Otherwise I can use google-recaptcha's JS API to make the component

Reply:
That's a good catch, you can use `react-google-recaptcha` as well. Since this is probably part of the site we will require a ReCaptcha it should be okay to leave it as it is. I'll leave the call to you if you do want to update it since this module still supports v2.

Reply:
switched to `react-google-recaptcha`, it's probably not the best idea to use libraries that haven't been updated